stack.yaml

```diff
@@ -1,7 +1,10 @@
-resolver: lts-16.22
+resolver: nightly-2020-12-03
```

Comment:
I'm not sure what our stance is on nightly resolvers. @pbrisbin do you have a policy for Freckle's OSS projects and snapshot alignment?

Reply:
Mentioned in my top-level review: I like to keep the default `stack.yaml` on "latest LTS" but then also test with `nightly` on CI.